services/scanner.py

```
@@ -186,105 +153,50 @@ async def scan_website(
-                    async with curl_cffi.requests.AsyncSession(impersonate="chrome120", timeout=self.timeout) as client:
-                        response = await client.get(pinned_url, allow_redirects=False, headers={"Host": host_header})
+            
+            try:
+                import curl_cffi.requests
+                async with curl_cffi.requests.AsyncSession(impersonate="chrome120", timeout=self.timeout) as client:
+                    response = await client.get(_safe_url, allow_redirects=True)
+                    html = response.text
+                    headers = response.headers
+                    cookies = response.cookies
+                    status_code = response.status_code
+            except Exception as curl_e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)